tsconfig.json

```diff
@@ -1,67 +1,20 @@
 {
   "compilerOptions": {
     "target": "es6",
+    "lib": ["es2016"],
```

Comment:
I'm good with this if we can agree to standardize across all repos? I've been using `es2016` up to this point.